src/index.js

```diff
@@ -337,14 +337,15 @@ export { ShaderHandler } from './framework/handlers/shader.js';
 export { SpriteHandler } from './framework/handlers/sprite.js';
 export { TemplateHandler } from './framework/handlers/template.js';
 export { TextHandler } from './framework/handlers/text.js';
-export { TextureHandler, TextureParser } from './framework/handlers/texture.js';
+export { TextureHandler } from './framework/handlers/texture.js';
 export { TextureAtlasHandler } from './framework/handlers/texture-atlas.js';
 
 // FRAMEWORK / INPUT
 export { ElementInput, ElementInputEvent, ElementMouseEvent, ElementSelectEvent, ElementTouchEvent } from './framework/input/element-input.js';
 
 // FRAMEWORK / PARSERS
 export { JsonStandardMaterialParser } from './framework/parsers/material/json-standard-material.js';
+export { TextureParser } from './framework/parsers/texture/texture.js';
```

Comment:
Not sure we need to expose it, it seems the class and its descendants are all `@ignore`.

Reply:
Yea i wasnt sure since it was exposed before.

Reply:
Might be used in the editor thats the ony thing I can think of?